package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@interledger/rafiki",
-  "version": "1.1.1-alpha",
+  "version": "1.0.0",
```

Comment:
not `-alpha.1` or similar?

Reply:
Yes, you are right. We should actually have a script that does that automatically when tagging